controllers/common/types.go

```diff
@@ -55,15 +55,15 @@ func (r *Reconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
 	chaos := r.Object()
 	if err = r.Get(ctx, req.NamespacedName, chaos); err != nil {
 		r.Log.Error(err, "unable to get chaos")
-		return ctrl.Result{}, nil
+		return ctrl.Result{}, err
```

Comment:
I think it will return to the entry reconciler such as `iochaos_controller.go`. Inside it, we just return `nil` such that it will work as expected. Am I misunderstanding this ? @cwen0 @zhouqiang-cl

Reply:
@AngleNet Sorry! this is my fault, you are right.

Reply:
That's OK. Will resolve this conversation